core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/wait.yml

```diff
@@ -25,7 +25,7 @@
   shell: kubectl get pods --all-namespaces -o json
   register: output
   until: output.stdout|from_json|json_query('items[*].status.phase')|unique == ["Running"]
-  retries: 150
+  retries: 600 # 20min
```

Comment:
Won't be those comments misleading if someone will forget to change number of minutes after changing retries?